dpnp/tests/third_party/cupy/linalg_tests/test_product.py

```diff
@@ -40,9 +40,21 @@
 )
 class TestDot(unittest.TestCase):
 
+    # Avoid overflow
+    skip_dtypes = {
+        (numpy.int8, numpy.int8),
+        (numpy.int8, numpy.uint8),
+        (numpy.uint8, numpy.uint8),
+    }
+
     @testing.for_all_dtypes_combination(["dtype_a", "dtype_b"])
     @testing.numpy_cupy_allclose(type_check=has_support_aspect64())
     def test_dot(self, xp, dtype_a, dtype_b):
+        if (dtype_a, dtype_b) in self.skip_dtypes or (
+            dtype_b,
+            dtype_a,
+        ) in self.skip_dtypes:
+            pytest.skip("avoid overflow")
```

Comment:
Why don't numpy has such kind of issue and it works for it?

Reply:
Most of the cases relate to `int8` and `uint8`. Probably it would be better to add new keyword `no_int8` to `for_all_dtypes` to cover that?